packages/mdc-button/_variables.scss

```diff
@@ -16,3 +16,6 @@
 
 $mdc-button-height: 36px;
 $mdc-dense-button-height: 32px;
+
+$mdc-unfilled-button-ripple-opacity: .16;
```

Comment:
You should reference these in our demo.scss as well.

Reply:
Good point!